Devex 69

gsaray opened the pull request:
## Changes
- update search icon
- Type `F` focuses on search input
- Type `C` focuses on component search input (this change was made in styleguidist)
- Update search placeholder

## Jira Tickets:
- https://mobify.atlassian.net/browse/DEVEX-69

## How to Test
- tab 1, npm link
- tab 2, Switch to `amp-sdk` and checkout `use-docs-theme` branch
- run `npm link documentation-theme`
- tab 1, `npm run sass`
- tab 2, `npm `i then `npm run docs:dev`
-  Test the changes

fractaltheory commented on layouts/base.pug:
```diff
@@ -135,6 +135,15 @@ html(lang='en')
+                    }
+                    onkeyup = function(e){
+                        var searchInput = document.getElementById('gsc-i-id1');
+                        if(e.keyCode==70 && e.target.tagName !== 'INPUT'){
```
Additionally, I think we also want to prevent cases where `tagName === 'TEXTAREA'` since we have that ProdPad widget now.

Nitpick: spaces before and after the `===` and before the opening code block `{` character.

gsaray replied:
good catch 🙏

gsaray commented:
I have addressed your feedbacks.
I don't think having a keypress on mobile mode makes sense since the keyboard is not visible unless an input is focused. Let me know if you think otherwise
